torch/_dynamo/variables/tensor.py

```diff
@@ -788,6 +791,13 @@ def _register_hook_trampoline(tensor):
                 ),
                 **options,
             )
+            if is_inplace_op:
+                assert (
+                    ret.as_proxy().node.meta["example_value"]
```

Comment:
Assert may be too strong. Running branch if the example_values are identical may be the solution if tests fail.

Reply:
Had to do this. Some non-inplace ops collide into the name condition.